packages/http-client-csharp/package.json

```diff
@@ -34,12 +34,16 @@
     "test:ci": "vitest run --coverage --reporter=junit --reporter=default",
     "lint": "eslint . --max-warnings=0",
     "lint:fix": "eslint . --fix",
-    "format": "pnpm -w format:dir packages/http-client-csharp"
+    "format": "pnpm -w format:dir packages/http-client-csharp",
+    "extract-api": "npx api-extractor run --local --verbose"
   },
   "files": [
     "dist/**"
   ],
   "dependencies": {
+    "@azure-tools/ts-genapi": "^1.0.7",
+    "@microsoft/api-extractor-model": "^7.28.2",
+    "js-tokens": "^8.0.0",
```

Comment:
are those not supposed to be dev dependencies

Reply:
are they even used at all? I see `"@microsoft/api-extractor": "^7.40.3",` as needed for the `api-extractor` command but not sure what those do